source/assets/stylesheets/_accessibility.scss

```diff
@@ -65,6 +65,11 @@ a:focus {
 .skiplink {
   position: absolute;
   left: -9999em;
+
+  /* Default link colour doesn't have enough contrast against $focus-colour */
+  &:focus, &:visited {
+    color: $black;
```

Comment:
This should probably be $text-colour (even though it maps to the same hex colour).